recipes/openmm/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "openmm" %}
+{% set version = "7.4.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cb7ddd923d6f402cf2723b3c14509915abad328505e8ce401ac3131706dabeff
+
+build:
+  number: {{ build }}
+  skip: True  # [py2k or win]
+  missing_dso_whitelist:
+  - "*/libcuda.*"  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cmake
+    - jom  # [win]
+
+  host:
+    - python
+    - fftw
+    - swig
+    - pthread-stubs
+    - lxml
+    - numpy
+    - cython
+    # needed for Python wrappers
+    - doxygen 1.8.14
+    # OpenCL ICD
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [osx or win]
+
+  run:
+    - python
+    - fftw
+    - numpy
+    - cython
```

Comment:
Why is `cython` needed at run time?

Reply:
No reason I know of!

Reply:
Let's go ahead and drop it then.

```suggestion
```

Reply:
Do you agree @jchodera? This was copied from the `omnia` recipe, so I don't know if there's a specific reason to keep it or was simply blindly inherited from previous versions.

Reply:
That was probably an accident.